core/controllers/profile.py

```diff
@@ -89,8 +89,15 @@ def get(self, username):
             edited_exp_summary_dicts = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     user_contributions.edited_exploration_ids))
+            ### function to determine whether current user is same as user on page ###
```

Comment:
Ah right. Would you like me to remove it? I was using that so I knew where I put the code and didn't scroll past it on accident